amsd/models.py

```diff
@@ -47,10 +53,36 @@ class holder_file(Base):
     pk = Column(Integer, primary_key=True)
     name = Column(Unicode)
 
+@implementer(amsd_interfaces.Isem_domain)
+class sem_domain(Base):
+    pk = Column(Integer, primary_key=True)
+    name = Column(Unicode)
+
+@implementer(amsd_interfaces.Isource_type)
+class source_type(Base):
+    pk = Column(Integer, primary_key=True)
+    name = Column(Unicode)
+
+@implementer(amsd_interfaces.Itechnique)
+class technique(Base):
+    pk = Column(Integer, primary_key=True)
+    name = Column(Unicode)
+
+@implementer(amsd_interfaces.Ikeyword)
+class keyword(Base):
+    pk = Column(Integer, primary_key=True)
+    name = Column(Unicode)
```

Comment:
See comment above (no need for interfaces). Also, controlled vocabularies can almost always be covered by the fields of [`IdNameDescriptionMixin`](https://github.com/clld/clld/blob/1026e664c73424c2980137ce1e397e91579e99ed/src/clld/db/models/_mixins.py#L18).